.github/workflows/ansible-lint.yaml

```diff
@@ -0,0 +1,21 @@
+# https://github.com/openstack-k8s-operators/osp-director-dev-tools/blob/dae8a937bf56a3d50e4a1ffa60b47a485b5baa22/.github/workflows/ansible-lint.yaml#L2
```

Comment:
remove this reference doesn't make sense any more. for one we are not using the ansible-lint-action action